package.json

```diff
@@ -2,14 +2,20 @@
   "name": "baby-sharks-room",
   "version": "0.0.0",
   "private": true,
+  "main": "server.js",
   "scripts": {
-    "start": "node ./bin/www",
-    "dev": "nodemon ./bin/www"
+    "client-install": "cd client && npm install",
+    "start": "node server.js",
```

Comment:
Please put back bin/www back as this hides the actual starting js file. This yields more abstraction.

www should have the port set up. Listening port can be removed in server.js itself. The correct reference in www

```
var app = require('../server');
```